routes/api.php

```diff
@@ -31,7 +31,7 @@
     Route::post('/logout', [AuthController::class, 'logout'])->middleware('auth:sanctum');
     Route::get('/track/{type}/leaderboard', [ParticipantController::class, 'leaderboard'])->middleware(['auth:sanctum', 'verified']);
 
-    Route::prefix('admin')->middleware(['auth:sanctum', 'hasRole:admin'])
+    Route::prefix('admin')->middleware(['auth:sanctum', 'hasRole:admin', 'verified'])
```

Comment:
I don't think we should add the `verified` middleware to the admin. It is something we control always.